### ISSUE TITLE
JSONFieldSchemaValidator: Django cannot serialize into migration files

### ISSUE THREAD

**Issue by JhonFrederick** [wblxyxolbkhv/django-json-schema-validator#1]
The following error occurs when the **makemigrations** command is applied when `JSONFieldSchemaValidator ` is used

```
ValueError: Cannot serialize: <django_json_field_schema_validator.validators.JSONFieldSchemaValidator object at 0x7f5c363b18e0>
There are some values Django cannot serialize into migration files.
```

According to  [documentation](https://docs.djangoproject.com/en/3.2/ref/validators/) is necessary a `deconstruct` method or  add `@deconstructible` in the class.

**Comment by wblxyxolbkhv** [comment]
Thank you a lot for reporting! You are totally right, it's my bad. I will fix it soon.

**Comment by wblxyxolbkhv** [comment]
Fixed in https://github.com/wblxyxolbkhv/django-json-field-schema-validator/pull/2 and released in version 0.0.2. Thanks for reporting again!
